cli/endpoints/online/managed/vnet/test_scoring/modules/vm.bicep

```diff
@@ -0,0 +1,99 @@
+param location string
+
+@description('VM resource name')
+param vmName string
+
+@description('Tags to add to the resources')
+param tags object = {}
+
+// random user name and sshk public key. The VM is not private IP enabled and accessed only using "az vm run-command invoke"
+param adminUserName string = 'd3F4gfddfE4'
```

Comment:
any random generator for username? (Unlike passwd, this need not be meet certain requirements)

Reply:
bicep does not seem to have any random key generator. The admin name can never be used as we are not setting a password. Is it ok?